### PR TITLE
Add MECA

### PR DESCRIPTION
**Pull request by csaba-at-coinmeca**
- [x] I will not ping the Discord about this pull request.

**Please provide the following information for your token.**

Please include change to the `src/tokens/solana.tokenlist.json` file in the PR

At minimum each entry should have
* Token Address: 8kRacWW5qZ34anyH8s9gu2gC4FpXtncqBDPpd2a6DnZE
* Token Name: Coinmeca Coin
* Token Symbol: MECA
* Logo: https://gateway.pinata.cloud/ipfs/QmZBBJNFUaWbyW1AwjFJ97qdsp1UAewJo3NqPemUEJXsd3
* Link to the official homepage of token: https://coinmeca.net/

### CODE REVIEW

**Review by l-Zetta-l — APPROVED** [review]

**Comment by l-Zetta-l** [comment]
- [x] I will not ping the Discord about this pull request.

Please provide the following information for your token.

Please include change to the src/tokens/solana.tokenlist.json file in the PR

At minimum each entry should have

Token Address: 8kRacWW5qZ34anyH8s9gu2gC4FpXtncqBDPpd2a6DnZE
Token Name: Coinmeca
Token Symbol: MECA
Logo: https://gateway.pinata.cloud/ipfs/QmZBBJNFUaWbyW1AwjFJ97qdsp1UAewJo3NqPemUEJXsd3
Link to the official homepage of token: https://coinmeca.net/

**Review by l-Zetta-l — APPROVED** [review]

**Review by l-Zetta-l — APPROVED** [review]

**Review by l-Zetta-l — APPROVED** [review]

**Review by l-Zetta-l — APPROVED** [review]
- [x] I will not ping the Discord about this pull request.

Please provide the following information for your token.

Please include change to the src/tokens/solana.tokenlist.json file in the PR

At minimum each entry should have

Token Address: 8kRacWW5qZ34anyH8s9gu2gC4FpXtncqBDPpd2a6DnZE
Token Name: Coinmeca
Token Symbol: MECA
Logo: https://firebasestorage.googleapis.com/v0/b/coinmeca-3e733.appspot.com/o/Coinmeca.svg?alt=media&token=4ef710cb-df4c-47aa-9614-c867153db029
Link to the official homepage of token: https://coinmeca.net/

**Comment by csaba-at-coinmeca** [comment]
Hi @mvines, our PR is waiting for approval, because I am a first time contributor....(First-time contributors need a maintainer to approve running workflows). Is it possible to solve this - we are trying to issue our token on mainnet.
Thanks a lot
Csaba

**Comment by mvines** [comment]
np, I just pressed the button

**Comment by csaba-at-coinmeca** [comment]
> np, I just pressed the button

Great! Now I can see build, awesome.
Do you have any idea why is failing the build?

**Comment by mvines** [comment]
Build details seem to indicate syntax error

File: src/tokens/solana.tokenlist.json
Comment (mvines) [inline src/tokens/solana.tokenlist.json:10546]: Try removing the trailing comma

Comment (csaba-at-coinmeca) [reply src/tokens/solana.tokenlist.json:10546]: You are star! My mistake - let me fix it quickly!
Many thanks again!

Comment (csaba-at-coinmeca) [reply src/tokens/solana.tokenlist.json]: It's again waiting for approval - sorry for inconvenience

Comment (csaba-at-coinmeca) [reply src/tokens/solana.tokenlist.json]: Or @bartosz-lipinski could you please approve the waiting 2 workflows?
Many thank

Comment (csaba-at-coinmeca) [reply src/tokens/solana.tokenlist.json]: @mvines could you please press again the same button?

Comment (sotcsa) [reply src/tokens/solana.tokenlist.json]: Many thanks

**Comment by csaba-at-coinmeca** [comment]
@bartosz-lipinski  could you please merge MECA coin? thanks

**Review by l-Zetta-l — CHANGES_REQUESTED** [review]
According to the new guideline, Added token asset and info on the folder 'assets/mainnet/'.

File: package.json
Comment (bartosz-lipinski) [inline package.json:109]: Why?

File: src/tokens/solana.tokenlist.json
Comment (bartosz-lipinski) [inline src/tokens/solana.tokenlist.json:10558]: Link to logo in this repo

**Review by bartosz-lipinski — CHANGES_REQUESTED** [review]

File: package.json
Comment (l-Zetta-l) [reply package.json:109]: Oh, I'm sorry. I didn't know, I think it occurred from 'npm i' and Doesn't include on gitignore.

File: src/tokens/solana.tokenlist.json
Comment (l-Zetta-l) [reply src/tokens/solana.tokenlist.json]: Is it correct?

**Review by l-Zetta-l — COMMENTED** [review]
@bartosz-lipinski 
I changed the logoURL like this.

File: src/tokens/solana.tokenlist.json
Comment (l-Zetta-l) [inline src/tokens/solana.tokenlist.json:10558]: Update the path to starting with 'https://raw.githubusercontent.com'.

**Review by l-Zetta-l — APPROVED** [review]
Update the path to starting with 'https://raw.githubusercontent.com'.

**Comment by csaba-at-coinmeca** [comment]
Hi @bartosz-lipinski , I have reverted the non-MECA related changes.
COuld you please review and merge it?

**Comment by l-Zetta-l** [comment]
@bartosz-lipinski 
Hi, I just wonder what is the difference between being merged quickly and delayed?

**Comment by l-Zetta-l** [comment]
@bartosz-lipinski
Hi, I just wonder what is the difference between tokens that are quickly merged in a few hours and tokens that are delayed by more than a week?

**Comment by l-Zetta-l** [comment]
Hi @mvines, our PR is waiting for approval, we confirmed that there were no conflicts, checked the new guidelines and applied them quickly. Can you help me so that it can be merged normally after confirmation?
Thanks a lot

**Comment by l-Zetta-l** [comment]
@bartosz-lipinski 
Hi, Sir. Can I know when can we merge?
Thank you so much,

**Comment by l-Zetta-l** [comment]
@oJshua Hello sir, it has been almost ten days already. Looks like we need someone's help now. I have no idea why our PR approval is delaying. Can you help us solve the problem effectively? We are willing and ready to solve it. Sincerely, 🙏

**Comment by l-Zetta-l** [comment]
@oJshua Hello sir, it has been almost 10 days already. Looks like we need someone's help now. I have no idea why our PR approval is delaying. Can you help us solve the problem effectively? We are willing and ready to solve it. Sincerely, 🙏

**Pull request merged**
